### ISSUE TITLE
Support for `Int256`

### ISSUE THREAD

**Issue by gd87429** [ClickHouse/clickhouse-rs#52]
Wasn't sure whether to comment on https://github.com/loyd/clickhouse.rs/issues/48 or not. Do you have any suggestions on how to handle Clickhouse's `Int256` in Rust?

**Comment by gd87429** [comment]
@loyd There's convenient [U256](https://docs.rs/ethers-core/latest/ethers_core/types/struct.U256.html) and [I256](https://docs.rs/ethers-core/latest/ethers_core/types/struct.I256.html) here btw.

**Comment by loyd** [comment]
These wrappers are convenient, but `ethers_core` is much bigger than just numeric types.

**Comment by gd87429** [comment]
@loyd You're right, it is much bigger. If not integrate these types into this lib, any advise on how to use outside of the lib?

**Comment by loyd** [comment]
You can implement `Serialize` and `Deserialize` (either by creating a custom wrapper type or using `serde(with)` attribute) in similar way as `U256` in https://github.com/loyd/clickhouse.rs/issues/48.
